auth/auth.go

```diff
@@ -79,6 +80,13 @@ func authenticate(tkn string) (string, error) {
 		return "", fmt.Errorf("token is expired")
 	}
 
+	blockedIds := strings.Split(os.Getenv("BLOCKED_CLIENT_IDS"), ",")
+	for _, id := range blockedIds {
+		if token.PublicKeyHex == id {
+			return "", fmt.Errorf("This client ID is blocked")
```

Comment:
what about users who are already authenticated to these IDs?

Reply:
@diracdeltas Whenever a sync request comes in, we will run the code path `Authorize()` -> `authenticate()`. https://github.com/brave/go-sync/blob/master/controller/controller.go#L36